services/viva-ms/lambdas/createVivaCase.js

```diff
@@ -152,3 +158,70 @@ async function putRecurringVivaCase(PK, workflowId, period) {
 
   return caseItem;
 }
+
+async function getUser(PK) {
+  const personalNumber = PK.replace('USER#', '');
+  const params = {
+    TableName: config.users.tableName,
+    Key: {
+      personalNumber,
+    },
+  };
+
+  const [error, dbResponse] = await to(dynamoDB.call('get', params));
+  if (!dbResponse) {
+    console.error('(cases-api) DynamoDb query on users table failed', error);
+    return;
+  }
```

Comment:
I would rather see that we throw this error than return nothing at all. the console.error can be used in an catch further upstream.